### PR TITLE
Project settings tests

### PR DESCRIPTION
**Pull request by JeanneSon**
## Description
Projects can now be created via the test control API and users can be added to a project via the test control API.

The following tests have been converted to playwright:

Bellows E2E Project Settings app
✓ Normal user cannot access projectSettings to a project of which the user is a member
✓ System Admin can manage project
✓ Manager cannot view delete tab if not owner
✓ Manager can delete if owner

The following tests have not been converted due to redundancy
✓ confirm Manager is not owner of test project
✓ confirm Manager is owner of fourth project

For details/remarks regarding converting the above tests, refer to the spreadsheet; https://docs.google.com/spreadsheets/d/1w9wjfm-02QHLhv0ObPAKKlqnDWeag379GIXwOcRU4W0/edit#gid=0

### Type of Change
E2E test

## Checklist

- [x] I have performed a self-review of my own code
- [x] I have reviewed the title/description of this PR which will be used as the squashed PR commit message
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/angular-app/languageforge/lexicon/settings/project-settings.component.html [hunk 1]
```diff
@@ -27,7 +27,7 @@
                     </div>
                     <div class="form-group row">
                         <label class="col-form-label col-12">Project Owner</label>
-                        <div class="controls col-12 notranslate">
+                        <div id="e2e-test-project-owner" class="controls col-12 notranslate">
```
Comment (JeanneSon) [inline src/angular-app/languageforge/lexicon/settings/project-settings.component.html:30]: As it was particularly difficult to write a locator for this element, I added an id to the HTML (according to [this article](https://github.com/goldbergyoni/javascript-testing-best-practices#-%EF%B8%8F-32-query-html-elements-based-on-attributes-that-are-unlikely-to-change) recommended by @longrunningprocess)

File: test/e2e/projects-settings.spec.ts [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    for (const project of projects) {
+      const projectId = await initTestProject(request, project.code, project.name, admin.username, [member.username]);
+      project.id = projectId;
+    }
+    await addUserToProject(request, projects[0].code, manager.username, 'manager');
+    project4.id = await initTestProject(request, project4.code, project4.name, manager.username, []);
+  });
+
+
+  // test if can change project name
```
Comment (JeanneSon) [inline test/e2e/projects-settings.spec.ts:51]: Should we also test whether one can change the name of a project?

Comment (megahirt) [reply test/e2e/projects-settings.spec.ts:51]: I think testing that you can change the name of the project would be nice, but not required for this PR.

**Review by megahirt — APPROVED** [review]
Everything I've browsed looks good to me!  Keep up the great work!

**Pull request merged**